docs/modules/operation/pages/bsm/business-service-hierarchy.adoc

```diff
@@ -1,15 +1,15 @@
 
-= Business Service Hierarchy
+= Business service hierarchy
 
-BS can depend on each other and together build a business service hierarchy.
-Visualize it using the Topology User Interface with the Business Services View.
-The operational status of a BS is ultimately calculated from alarms and their serverity.
-To define the class of alarms a reduction key is used and is represented as an edge of a BS.
+Business services can depend on each other and together build a business service hierarchy.
+Visualize the hierarchy using the Topology User Interface with the business services view.
+The operational status of a business service is ultimately calculated from alarms and their serverity.
+Ues a To define the class of alarms a reduction key is used and is represented as an edge of a business service.
```

Comment:
Honestly, this sentence seems out of place in the paragraph and might be better off removed from here.
```suggestion
Use a reduction key to define a class of alarms as an edge to a business service.
```